### ISSUE TITLE
Entrypoint uses same name as upstream

### ISSUE THREAD

**Issue by Nolij** [calmilamsy/ModMenu#5]
This mod uses the same `modmenu` entrypoint that upstream does, which makes it incompatible with my multi-version mod, [Zume](https://github.com/Nolij/Zume). This is not easily addressed on my side. I'm not sure why the same entrypoint name was used, but I think it'd make more sense to use a different name to avoid incompatibility issues like this. As is, I can't fix the incompatibility (which currently causes a crash), or explicitly add compatibility for this fork without doing some very ugly hacks that just create work.

TL;DR I'm requesting that you change the entrypoint name to something different from upstream. Perhaps `modmenubeta`?
